3-UsingAPIs/Week2/homework/ex1-programmerFun/index.js

```diff
@@ -15,29 +15,44 @@
    url with `.shx`. There is no server at the modified url, therefore this 
    should result in a network (DNS) error.
 ------------------------------------------------------------------------------*/
-function requestData(url) {
+async function requestData(url) {
   // TODO return a promise using `fetch()`
+  const response = await fetch(url);
+  const json = await response.json();
+  return json;
 }
 
 function renderImage(data) {
   // TODO render the image to the DOM
-  console.log(data);
+  const image = data.img
```

Comment:
There is a semicolon missing here, but I think we solved in your VSCode config already.